.eslintrc.json

```diff
@@ -0,0 +1,51 @@
+{
+  "extends": "airbnb-base",
+  "parserOptions": {
+    "ecmaVersion": 8,
+    "sourceType": "module"
+  },
+  "rules": {
+    "consistent-return": 0,
+    "max-len": [2,
+      {
+        "code": 400
+      }
+    ],
+    "import/no-dynamic-require": 0,
+    "import/prefer-default-export": 0,
+    "no-console": 0,
+    "no-use-before-define": ["error",
+      {
+        "functions": true,
+        "classes": false
+      }
+    ],
+    "no-param-reassign": ["error",
+      {
+        "props": true,
+        "ignorePropertyModificationsFor": ["poolAction"]
+      }
+    ],
+    "no-unused-vars": ["error",
+      {
+        "args": "none", // "all" for everything
+        "caughtErrors": "none"
+      }
+    ],
+    "class-methods-use-this": [0,
+      {
+        "exceptMethods": ["foo"] // Not effective since it's 0
```

Comment:
"exceptMethods": ["foo"] // Not effective since it's 0

Let's remove this line since ineffective? 

On second thought, let's remove this block "class-methods-use-this", meaning class method has to use this. I don't know why it's disabled, but If you find it's annoying you can add it back in future.

Reply:
done, removed all unnecessary rules